pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -151,9 +152,14 @@
         "1.0.2",
         "1.1.0",
         "2.0.0",
-        "2.1.0"
-      ],
-      "BaselineVersion": "2.0.0",
+        "2.1.0",
+        "2.1.1",
+        "2.1.2",
+        "2.1.3",
+        "2.1.4",
+        "2.2.0"
+      ],
+      "BaselineVersion": "2.2.0",
```

Comment:
@ericstj should I still update baseline version for Microsoft.NETCore.Platforms/Targets package as well? Does it even matter?